README.md

```diff
@@ -96,13 +96,20 @@ Then, open a command prompt in the new `ixmp` directory and type
 
 ## Configure Model Files
 
-By default, the model files are stored with your installation. If you would like
-to store them some place locally, open a command prompt and type
+By default, the model files are stored with your installation. Many users will
```

Comment:
Does it make sense to add an example path here - I would not have known what "stored with your installation" means a little while ago...
Also, should this mention GAMS - again, it might not be obvious what "model files" are...